include/linker/common-ram.ld

```diff
@@ -62,6 +62,13 @@
 		_k_sem_list_end = .;
 	} GROUP_DATA_LINK_IN(RAMABLE_REGION, ROMABLE_REGION)
 
+	SECTION_DATA_PROLOGUE(_sys_ring_buf_area, (OPTIONAL), SUBALIGN(4))
+	{
+		_sys_ring_buf_list_start = .;
+		KEEP(*(SORT_BY_NAME("._sys_ring_buf.static.*")))
```

Comment:
Did you test this?
There is currently no mechanism to put ring buffers in this section, see definition of SYS_RING_BUF_DECLARE_POW2 and SYS_RING_BUF_DECLARE_SIZE.